src/main/java/de/adesso/kicker/notification/Message/MessageType.java

```diff
@@ -0,0 +1,16 @@
+package de.adesso.kicker.notification.Message;
+
+public enum MessageType {
+
+    MESSAGE_DECLINED("notification.message.declined");
+
+    String messageType;
+
+    private MessageType(String messageType) {
```

Comment:
I remember as Mr. Hadrys always told us to set the modifier of an enum to private. Well that's not necessary, intelliJ told me `Modifier 'private' is redundant for enum constructors` to I googled it and found out that 

> If no access modifier is specified for the constructor of an enum type, the constructor is private.

Just a little Sidefact. I didn't know that too. But please keep it like that just wanted to share you my discovery :D